src/MCPServer/lib/archivematicaMCP.py

```diff
@@ -260,6 +260,74 @@ def cleanupOldDbEntriesOnNewRun():
 
 
+def created_shared_directory_struture():
+    dirs = (
+        "arrange",
+        "completed",
+        "completed/transfers",
+        "currentlyProcessing",
+        "DIPbackups",
+        "failed",
+        "rejected",
+        "sharedMicroServiceTasksConfigs",
+        "sharedMicroServiceTasksConfigs/createXmlEventsAssist",
+        "sharedMicroServiceTasksConfigs/generateAIP",
+        "sharedMicroServiceTasksConfigs/generateAIP/bagit",
+        "sharedMicroServiceTasksConfigs/processingMCPConfigs",
+        "sharedMicroServiceTasksConfigs/transcoder",
+        "sharedMicroServiceTasksConfigs/transcoder/defaultIcons",
+        "SIPbackups",
+        "tmp",
+        "watchedDirectories",
+        "watchedDirectories/activeTransfers",
+        "watchedDirectories/activeTransfers/baggitDirectory",
+        "watchedDirectories/activeTransfers/baggitZippedDirectory",
+        "watchedDirectories/activeTransfers/Dspace",
+        "watchedDirectories/activeTransfers/maildir",
+        "watchedDirectories/activeTransfers/standardTransfer",
+        "watchedDirectories/activeTransfers/TRIM",
+        "watchedDirectories/approveNormalization",
+        "watchedDirectories/approveSubmissionDocumentationIngest",
+        "watchedDirectories/quarantined",
+        "watchedDirectories/SIPCreation",
+        "watchedDirectories/SIPCreation/completedTransfers",
+        "watchedDirectories/SIPCreation/SIPsUnderConstruction",
+        "watchedDirectories/storeAIP",
+        "watchedDirectories/system",
+        "watchedDirectories/system/autoProcessSIP",
+        "watchedDirectories/system/autoRestructureForCompliance",
+        "watchedDirectories/system/createAIC",
+        "watchedDirectories/system/reingestAIP",
+        "watchedDirectories/uploadDIP",
+        "watchedDirectories/uploadedDIPs",
+        "watchedDirectories/watchedDirectories",
+        "watchedDirectories/watchedDirectories/system",
+        "watchedDirectories/watchedDirectories/system/autoProcessSIP",
+        "watchedDirectories/workFlowDecisions",
+        "watchedDirectories/workFlowDecisions/compressionAIPDecisions",
+        "watchedDirectories/workFlowDecisions/createDip",
+        "watchedDirectories/workFlowDecisions/createTree",
+        "watchedDirectories/workFlowDecisions/examineContentsChoice",
+        "watchedDirectories/workFlowDecisions/extractPackagesChoice",
+        "watchedDirectories/workFlowDecisions/metadataReminder",
+        "watchedDirectories/workFlowDecisions/quarantineTransfer",
+        "watchedDirectories/workFlowDecisions/selectFormatIDToolIngest",
+        "watchedDirectories/workFlowDecisions/selectFormatIDToolTransfer",
+        "www",
+        "www/AIPsStore",
+        "www/AIPsStore/transferBacklog",
+        "www/AIPsStore/transferBacklog/arrange",
+        "www/AIPsStore/transferBacklog/originals",
+        "www/DIPsStore"
+    )
+    for d in dirs:
+        d = os.path.join(django_settings.SHARED_DIRECTORY, d)
```

Comment:
it's a good point lower29, and sevein I agree - the exact structure of the shared directory structure is tied to the version of Archivematica, not to the version of the storage service.  

In the long run, it is probably worth exploring whether or not we really want to have a shared directory structure - the whole thing is based on the assumption that we need a shared filesystem for all the components of Archivematica to work together.  This comes from running all of it on a single machine.  In a container based deployment, with an object storage system available, it is less clear that a shared filesystem is a good idea.

This conversation is important, but I think out of scope for this PR. I think the changes proposed here are good, and it is worth continuing the conversation elsewhere for further changes.